doc-content/enterprise-only/springboot/bus-app_con.adoc

```diff
@@ -0,0 +1,25 @@
+[id='bus_app_{context}']
+= {PRODUCT} business application
+
+{PRODUCT} business applications are flexible, UI-agnostic logical groupings of individual services that provide certain business capabilities. They are usually deployed separately and can be versioned individually. A complete business application enables a domain to achieve specific business goals, for example order management or accommodation management. After you create and configure your business application, you can deploy it to an existing service through {CENTRAL} or to the cloud through OpenShift. 
+
+Business applications can have the following capabilities:
+
+* Persistence
+* Messaging
+* Transactions
+* Business processes and business rules
+* Planning solutions
+
+Business applications can contain one or more of the following projects and UI module and more than one project of the same type:
+
+* Business assets (kjar): Contains business processes, rules, and forms and are easily imported into {CENTRAL}. 
+* Data model: Data model projects provide common data structures that are shared between the service projects and business assets projectss. This enables proper encapsulation, promotes reuse, and reduces shortcuts. Each service project can expose its own public data model.
+* Service: A deployable project that provides the actual service with various capabilities. It includes the business logic that operates your business. In most cases, a service  project includes business assets and data model projects. A business application can split services into smaller component service projects for better manageability. 
+* UI modules: Use one module for each service embedded in the service project or use a federated UI module by creating a seperate UI project. COMMENT: Where do I add this in the web site?
```

Comment:
we don't provide any UI modules/projects as we want to stay away from an UI frameworks. So this is completely up to developers to build with frameworks they select